src/scripts/components/story-footer/story-footer.styl

```diff
@@ -6,3 +6,10 @@
   height: emCalc(72px)
   height: $footerHeight
   background-color: $darkGrey1
+
+.fadeOutFooter
+  opacity: 0
+  transition: opacity 0.4s ease-in-out
+
+  &:hover
```

Comment:
I'm not sure if it should work this way. In my understanding the header and footer should only be visible when the mouse has moved and disappear after a few seconds when no more interactions happened. Like in the Earth View extension.
Also I thought we they should slide in/out instead of fading. Could you check that gain with Patrick please.

Reply:
Talked to him, he said that showing the items when mouse is moving would be cool. 👍